java/clients/src/main/java/sleeper/clients/QueryClient.java

```diff
@@ -121,7 +121,7 @@ public static void main(String[] args) throws StateStoreException, ObjectFactory
         AmazonDynamoDB dynamoDB = buildAwsV1Client(AmazonDynamoDBClientBuilder.standard());
         InstanceProperties instanceProperties = ClientUtils.getInstanceProperties(amazonS3, args[0]);
 
-        QueryClient queryClient = new QueryClient(amazonS3, instanceProperties, dynamoDB, new Configuration());
+        QueryClient queryClient = new QueryClient(amazonS3, instanceProperties, dynamoDB, getConfigurationForClient());
```

Comment:
Should this configuration be applied to all the clients?

Reply:
I don't think it is needed. They seem to working as they are.